--- a/configure.ac
+++ b/configure.ac
@@ -447,9 +447,6 @@ if test "$enable_py_vortex_support" = "y
       # check python_install_dir and pythondir maches
       python_dir_temp=`eval echo ${pythondir}`
       # check python_install_dir and pythondir maches
-      if test "$python_dir_check" != "ok" ; then
-          AC_MSG_ERROR([ERROR: pythondir variable is poiting to an unusable location: $python_dir_temp. Error detected: $python_dir_check. Fix your autoconf install or disable python building, for that, pass --disable-py-vortex to configure. This usually indicates a problem with automake package which provides the _AM_PATH_PYTHON_ macro.])
-      fi
       echo "Python install dir OK: $python_dir_temp"
    fi
 fi
